packages/grid/backend/grid/core/node.py

```diff
@@ -27,15 +29,23 @@
 node_type = get_node_type()
 node_name = get_node_name()
 
+node_side_type = get_node_side_type()
+enable_warnings = get_enable_warnings()
+
+
 if node_type == "gateway" or node_type == "network":
     worker = Gateway(
         name=node_name,
+        node_side_type=node_side_type,
         action_store_config=sql_store_config,
         document_store_config=mongo_store_config,
+        enable_warnings=enable_warnings,
     )
 else:
     worker = Domain(
         name=node_name,
+        node_side_type=node_side_type,
```

Comment:
We could add also in worker.py, as that is being used for the single container deployments.

Reply:
Fixed by: https://github.com/OpenMined/PySyft/pull/7916/commits/85569e72674e2b015af149cf24b471dfa785ca38